osv/ecosystems/hackage.py

```diff
@@ -0,0 +1,124 @@
+# Copyright 2021 Google LLC
```

Comment:
Could you please add some simple tests in a `hackage_test.py`, along the lines of whats in [pypi_test.py](https://github.com/google/osv.dev/blob/1838dfe41a1383261cd3402bdd962935414fa2b1/osv/ecosystems/pypi_test.py) or [rubygems_test.py](https://github.com/google/osv.dev/blob/1838dfe41a1383261cd3402bdd962935414fa2b1/osv/ecosystems/rubygems_test.py)?

Reply:
No problem.  I'll do it in the next day or so.